packages/muze/src/options.js

```diff
@@ -58,7 +58,8 @@ export default {
             typeCheck: 'constructor',
             typeExpected: 'Object',
             sanitization: (config, oldConfig) => {
-                const oldConf = mergeRecursive({}, oldConfig);
+                // Reset the configuration when null is passed
+                const oldConf = mergeRecursive({}, config === null ? {} : oldConfig);
```

Comment:
what happens if config is undefined?

Reply:
If config is undefined, then it will merge the old configuration. Only if config is null, then it will reset the configuration.